data_structures/linked_list/Reverse_Linked_List.c

```diff
@@ -0,0 +1,120 @@
+#include <stdio.h>
+#include <stdlib.h>
+struct node {
```

Comment:
we can write typedef struct node instead of only struct node 
so that we don't have to write struct again and again